src/bgw_policy/drop_chunks.h

```diff
@@ -9,10 +9,15 @@
 
 #include "catalog.h"
 #include "export.h"
+#include "chunk.h"
 
 typedef struct BgwPolicyDropChunks
 {
-	FormData_bgw_policy_drop_chunks fd;
+	int32 job_id;
+	int32 hypertable_id;
+	FormData_ts_interval older_than;
+	bool cascade;
+	CascadeToMaterializationOption cascade_to_materializations;
```

Comment:
I don't see why we need a 3-state cascade_to_materialization. The default should be false (like before).  And the user can over-ride it to true. Don't see why it is useful to have a null/unknown state.

Reply:
the default  should be NULL